Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -136,7 +136,7 @@
 "wikitext-upload-save-anonymously" = "Button text for publishing without logging in";
 "wikitext-upload-save-anonymously-warning" = "Button sub-text informing user or draw-backs of not signing in before saving wikitext. Parameters:\n* $1 - sign in button text";
 "wikitext-upload-save-terms-and-license" = "Button text for information about the Terms of Use and edit license. Parameters:\n* $1 - 'Terms of Use' link ([[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-name]])\n* $2 - license name link";
-"wikitext-upload-save-terms-name" = "This message is used as $1 in the message [[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-and-license]].\n{{Identical|Terms of use}}";
+"wikitext-upload-save-terms-name" = "This message is used in the message [[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-and-license]].\n{{Identical|Terms of use}}";
```

Comment:
@amrox This does seem like a case where a mis-match in number of "$" is acceptable... hmm... how about if we do this:

- make the test fail (as it does now) if the number of "$" in the QQQ value doesn't match the number of "$" in the EN value
- but don't fail if the EN value has no "$"

That way we'd enforce "$" being added to QQQ for any EN having "$", but we'd still be able to add "$" to QQQ in cases like the one you found...